### PR TITLE
Update teams-classes-with-canvas.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated steps on how to grant admin consent

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/5364

File: microsoft-365/lti/teams-classes-with-canvas.md [hunk 1]
```diff
@@ -28,9 +28,19 @@ Microsoft Teams classes is a Learning Tools Interoperability (LTI) app that help
 > - Only a single Microsoft tenant can be used for mapping users between Canvas and Microsoft.
 > - You will have to turn off School Data Sync (SDS) before using the Class Teams LTI in order to avoid duplication of groups.
 
-## Microsoft Office 365 Admin
+## Grant Admin Consent
```
Comment (JohanFreelancer9) [inline microsoft-365/lti/teams-classes-with-canvas.md]:
```suggestion
## Grant admin consent
```

File: microsoft-365/lti/teams-classes-with-canvas.md [hunk 2]
```diff
@@ -28,9 +28,19 @@ Microsoft Teams classes is a Learning Tools Interoperability (LTI) app that help
...
+
+Prior to managing the Microsoft Teams integration within Instructure Canvas, it is important to have Canvas’s **Microsoft-Teams-Sync-for-Canvas** Azure app approved by your institution’s Microsoft Office 365 admin in your Microsoft Azure tenant before completing the Canvas admin setup. You need to be a Global Administrator to perform these steps.
+
+1. Navigate to Azure Active Directory.
+
+2. Open Enterprise applications > Select the Application **Microsoft-Teams-Sync-for-Canvas**
```
Comment (JohanFreelancer9) [inline microsoft-365/lti/teams-classes-with-canvas.md]:
```suggestion
2. Open Enterprise applications and select the **Microsoft-Teams-Sync-for-Canvas** application.
```

File: microsoft-365/lti/teams-classes-with-canvas.md [hunk 3]
```diff
@@ -28,9 +28,19 @@ Microsoft Teams classes is a Learning Tools Interoperability (LTI) app that help
...
 
-Prior to managing the Microsoft Teams integration within Instructure Canvas, it is important to have Canvas’s **Microsoft-Teams-Sync-for-Canvas** Azure app approved by your institution’s Microsoft Office 365 admin in your Microsoft Azure tenant before completing the Canvas admin setup.
+3. Select Permissions and then click Grant admin consent
+
+4. Agree with the permissions the application requires and grant consent
+
+## Microsoft Office 365 Admin
```
Comment (JohanFreelancer9) [inline microsoft-365/lti/teams-classes-with-canvas.md]:
```suggestion
3. Select **Permissions** and then choose **Grant admin consent**.

4. Agree with the permissions the application requires and grant consent.

## Microsoft Office 365 admin
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@cichur Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@tiburd

File: microsoft-365/lti/teams-classes-with-canvas.md [hunk 4]
```diff
@@ -28,9 +28,19 @@ Microsoft Teams classes is a Learning Tools Interoperability (LTI) app that help
 > - Only a single Microsoft tenant can be used for mapping users between Canvas and Microsoft.
 > - You will have to turn off School Data Sync (SDS) before using the Class Teams LTI in order to avoid duplication of groups.
 
-## Microsoft Office 365 Admin
+## Grant admin consent
 
-Prior to managing the Microsoft Teams integration within Instructure Canvas, it is important to have Canvas’s **Microsoft-Teams-Sync-for-Canvas** Azure app approved by your institution’s Microsoft Office 365 admin in your Microsoft Azure tenant before completing the Canvas admin setup.
+Prior to managing the Microsoft Teams integration within Instructure Canvas, it is important to have Canvas’s **Microsoft-Teams-Sync-for-Canvas** Azure app approved by your institution’s Microsoft Office 365 admin in your Microsoft Azure tenant before completing the Canvas admin setup. You need to be a Global Administrator to perform these steps.
```
Comment (tiburd) [inline microsoft-365/lti/teams-classes-with-canvas.md]:
```suggestion
Before managing the Microsoft Teams integration within Instructure Canvas, it's important to have Canvas’s **Microsoft-Teams-Sync-for-Canvas** Azure app approved by your institution’s Microsoft Office 365 admin in your Microsoft Azure tenant before completing the Canvas admin setup. You need to be a Global Administrator to perform these steps.
```

File: microsoft-365/lti/teams-classes-with-canvas.md [hunk 5]
```diff
@@ -28,9 +28,19 @@ Microsoft Teams classes is a Learning Tools Interoperability (LTI) app that help
...
+
+1. Navigate to Azure Active Directory.
+
+2. Open Enterprise applications and select the **Microsoft-Teams-Sync-for-Canvas** application.
```
Comment (tiburd) [inline microsoft-365/lti/teams-classes-with-canvas.md]:
```suggestion
2. Open Enterprise applications, and then select the **Microsoft-Teams-Sync-for-Canvas** application.
```

File: microsoft-365/lti/teams-classes-with-canvas.md [hunk 6]
```diff
@@ -28,9 +28,19 @@ Microsoft Teams classes is a Learning Tools Interoperability (LTI) app that help
...
+
+3. Select **Permissions** and then choose **Grant admin consent**.
```
Comment (tiburd) [inline microsoft-365/lti/teams-classes-with-canvas.md]:
```suggestion
3. Select **Permissions**, and then select **Grant admin consent**.
```

File: microsoft-365/lti/teams-classes-with-canvas.md [hunk 7]
```diff
@@ -28,9 +28,19 @@ Microsoft Teams classes is a Learning Tools Interoperability (LTI) app that help
...
+
+4. Agree with the permissions the application requires and grant consent.
```
Comment (tiburd) [inline microsoft-365/lti/teams-classes-with-canvas.md]:
```suggestion
4. Agree with the permissions that the application requires, and then grant consent.
```

**Pull request merged**